src/foundry/foundry.js/hooks.d.ts

```diff
@@ -281,10 +288,10 @@ declare global {
       ) => boolean | void;
 
       /**
-       * @remarks This is called during the drop portion of a drag-and-drop event on a roll table.
-       * @param entity - the Document the table belongs to
-       * @param config - the RollTableConfig
-       * @param data   - the dropped data, already parsed as an object via JSON
+       * A hook event that fires when some useful data is dropped onto a RollTableConfig.
+       * @param table  - The RollTable
```

Comment:
The parameters in the function definition should be renamed according to the documentation.